CHANGELOG.md

```diff
@@ -13,7 +13,8 @@
 
 ### What's new?
 
-- Objects can export the `Ord` trait, allowing such objects to be ordered.
+- Objects can export the `Ord` trait, allowing such objects to be ordered ([#2583](https://github.com/mozilla/uniffi-rs/pull/2583)).
+- Objects can implement external traits ([#2430](https://github.com/mozilla/uniffi-rs/issues/2430))
```

Comment:
Is this new in this commit?  I thought it was more about records/enums.  If not and you just want to sneak in the doc update, I'm fine with that.  I just got a bit confused reading the change.

Reply:
no, it was just something I noticed was missing.